### ISSUE TITLE
Build fails with error : panic: runtime error: index out of range [0] with length 0

### ISSUE THREAD

**Issue by devscipline** [devspace-cloud/devspace#1097]
**What happened?**  
I randomly get the following error when building an image using kaniko v0.19.0 in which I run a Webpack build :

```
Running webpack ...

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/devspace-cloud/devspace/pkg/devspace/build/builder/kaniko.(*Builder).BuildImage.func2(0x0, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/pkg/devspace/build/builder/kaniko/kaniko.go:340 +0x1e25
k8s.io/kubectl/pkg/util/interrupt.(*Handler).Run(0xc001988f90, 0xc001adad20, 0x0, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/vendor/k8s.io/kubectl/pkg/util/interrupt/interrupt.go:103 +0x139
github.com/devspace-cloud/devspace/pkg/devspace/build/builder/kaniko.(*Builder).BuildImage(0xc0000fa690, 0xc001840690, 0x1f, 0xc001840660, 0x2f, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/runner/runners/2.169.0/work/devspace/devspace/pkg/devspace/build/builder/kaniko/kaniko.go:196 +0x522
github.com/devspace-cloud/devspace/pkg/devspace/build/builder/helper.(*BuildHelper).Build(0xc0011d0300, 0x60eaf60, 0xc0000fa690, 0x6180fa0, 0x74b7040, 0x0, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/pkg/devspace/build/builder/helper/helper.go:109 +0x29c
github.com/devspace-cloud/devspace/pkg/devspace/build/builder/kaniko.(*Builder).Build(0xc0000fa690, 0x6180fa0, 0x74b7040, 0xc00119e201, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/pkg/devspace/build/builder/kaniko/kaniko.go:96 +0x53
github.com/devspace-cloud/devspace/pkg/devspace/build.(*controller).Build(0xc0011da3f0, 0xc000fdad4b, 0x6180fa0, 0x74b7040, 0x0, 0x0, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/pkg/devspace/build/build.go:132 +0x677
github.com/devspace-cloud/devspace/cmd.(*DeployCmd).Run(0xc000146540, 0x617e3e0, 0x750d3e0, 0xc000dea000, 0xc00027a230, 0x0, 0x5, 0x0, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/cmd/deploy.go:202 +0x12e9
github.com/devspace-cloud/devspace/cmd.NewDeployCmd.func1(0xc000dea000, 0xc00027a230, 0x0, 0x5, 0x0, 0x0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/cmd/deploy.go:67 +0x64
github.com/spf13/cobra.(*Command).execute(0xc000dea000, 0xc00027a1e0, 0x5, 0x5, 0xc000dea000, 0xc00027a1e0)
        /Users/runner/runners/2.169.0/work/devspace/devspace/vendor/github.com/spf13/cobra/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc000dd0000, 0x750d3e0, 0xc000dd0000, 0xc0006eff90)
        /Users/runner/runners/2.169.0/work/devspace/devspace/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/runner/runners/2.169.0/work/devspace/devspace/vendor/github.com/spf13/cobra/command.go:864
github.com/devspace-cloud/devspace/cmd.Execute()
        /Users/runner/runners/2.169.0/work/devspace/devspace/cmd/root.go:85 +0xae
main.main()
        /Users/runner/runners/2.169.0/work/devspace/devspace/main.go:17 +0x3e
```

When I try building the image again, without changing anything, it works.

**What did you expect to happen instead?**  
I expect the build to complete successfully.

**How can we reproduce the bug?** (as minimally and precisely as possible)  
You might reproduce it by running a Webpack build when building a docker image using kaniko v0.19.0

**Local Environment:**  
- DevSpace Version: 4.12.2
- Operating System: mac
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: google
- Kubernetes Version: v1.15.11-gke.3

/kind bug

**Comment by FabianKramm** [comment]
@devscipline thanks for reporting! I'll fix this right away!

**Comment by FabianKramm** [comment]
Should be fixed with v4.12.3
